src/analysis/pb_chance/mod.rs

```diff
@@ -8,7 +8,7 @@
 //! The PB chance is currently calculated through the Balanced PB algorithm. The
 //! PB chance is the percentile at which the Balanced PB algorithm finds the PB.
```

Comment:
I think we should update this explanation to mention skill curves instead of "the Balanced PB algorithm".